blocks/init/src/Blocks/components/modal/components/modal-editor.js

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import classnames from 'classnames';
+import { __ } from '@wordpress/i18n';
+import { outputCssVariables, getUnique, checkAttr, selector } from '@eightshift/frontend-libs/scripts';
+import manifest from './../manifest.json';
+import globalManifest from './../../../manifest.json';
+
+export const ModalEditor = (attributes) => {
+	const unique = useMemo(() => getUnique(), []);
+
+	const {
+		componentClass,
+		resources: { icon },
+	} = manifest;
+
+	const {
+		selectorClass = componentClass,
+		additionalClass,
+		blockClass,
+	} = attributes;
+
+	const modalUse = checkAttr('modalUse', attributes, manifest);
+	const modalExitButton = checkAttr('modalExitButton', attributes, manifest);
+	const modalContent = checkAttr('modalContent', attributes, manifest);
+	const modalId = checkAttr('modalId', attributes, manifest);
+
+	if (!modalUse) {
+		return null;
+	}
+
+	const modalClass = classnames([
+		selector(componentClass, componentClass),
+		selector(blockClass, blockClass, selectorClass),
+		selector(additionalClass, additionalClass),
+	]);
+
+	const modalOverlayClass = selector(componentClass, componentClass, 'overlay');
+	const modalDialogClass = selector(componentClass, componentClass, 'dialog');
+	const modalContentClass = selector(componentClass, componentClass, 'content');
+	const modalExitButtonClass = selector(componentClass, componentClass, 'close-button');
+
+	return (
+		<div className={modalClass} data-id={unique} id={modalId} aria-hidden="false">
+			{outputCssVariables(attributes, manifest, unique, globalManifest)}
```

Comment:
You're not using CSS variables in this block, so this inspires a question for the whole team - should we give data-ids and use `outputCssVariables` when bootstrapping blocks that don't use CSS variables at that point? (for future-proofing)

Or should we add CSS variable support to blocks only when they use them? Both approaches make sense, but seems that we should be consistent about this.

Reply:
if you are not using it remove it

Reply:
This is my bad
I'll remove it